index.html

```diff
@@ -29,5 +29,137 @@ <h1>Hey there. I'm Nilton<span> I'm a Full Stack web developer</span></h1>
           <li><a href="https://angel.co/u/nilton-segura"><img src="img/angellist.png"></a></li>
         </ul>         
     </div>
+    <a href="#"><img class="button" src="img/NormalButtonDisabled.png"></a>
+    <h2>My Recent Works</h2>
+    <img src="img/Indicator.png" alt="Litte divisor">
+    <div class="Works">
+      <div class="project">
```

Comment:
- [ ] Since the wrapper is to become a section, let's make this an `<article>` to make sure we respect the [HTML/CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) :recycle:

Reply:
Done. Thanks for your feedback. @GedeonTS